ytypes/util_schema.go

```diff
@@ -88,13 +88,11 @@ func validateListAttr(schema *yang.Entry, value interface{}) util.Errors {
 		return util.NewErrs(fmt.Errorf("schema %s ListAttr is nil", schema.Name))
 	}
 
-	var size int
-	if value == nil {
-		size = 0
-	} else {
+	var size uint64 = 0
```

Comment:
This can just be `var size uint64` - it'll be initialised with the zero-value anyway.